app/lib/report_helper.rb

```diff
@@ -497,9 +509,13 @@ def taxonomy_details(report, params)
 
     count_species_per_genus!(tax_2d)
 
-    # Compute aggregate scores
+    # Compute aggregate scores and all genera
+    all_genera = Set.new
```

Comment:
It's hard for me to determine if all_genera including all genera (including non-filtered) in the sample.